src/db/queries.ts

```diff
@@ -21,6 +25,17 @@ export async function createDiscussionThread(data: Prisma.ThreadUncheckedCreateI
   return await prisma.thread.create({ data })
 }
 
+export async function updateThreadTimeStamp(data: Prisma.ThreadUncheckedUpdateInput) {
```

Comment:
change the function name to `updateThread` and inside the function use the `data` directly

Reply:
done